site/pages/resources/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { Resources } from "components/pages/Resources";
+import { fetchCMSContent } from "lib/fetchCMSContent";
+import { loadTranslation } from "lib/i18n";
+import { GetStaticPropsContext } from "next";
+
+export async function getStaticProps(ctx: GetStaticPropsContext) {
+  try {
+    const documents = await fetchCMSContent("allDocuments");
+
+    const translation = await loadTranslation(ctx.locale);
+    if (!documents) {
+      throw new Error("No documents found");
+    }
+
+    return {
+      props: {
+        documents,
+        translation,
+      },
+      revalidate: 240,
+    };
+  } catch (e) {
+    console.error("RESOURCES getStaticProps error: ", e);
```

Comment:
in this case, if no documents are found something is gravely wrong... so I would `throw` the error again here, to crash the whole `getStaticProps` fn.

Only actual thrown errors are saved in my vercel logs, these `console.error` messages aren't stored, so it will be easier to debug if we throw again:

```suggestion
    throw e
```

Reply:
I see, all other pages do not throw either.
Either I will remove this throw again after Go Live
Or align all other pages

Reply:
Done